docs/whats-new-in-c8/whats-new-in-c8.md

```diff
@@ -53,7 +52,7 @@ To make the transition easier, Cylc 8 can run Cylc 7 workflows out of the box.
 ## Architecture
 [TOP](#whats-new-in-cylc-8)
 
-Cylc has been re-architected to support a remote web UI:
+Cylc has been re-architectured to support a remote web UI:
```

Comment:
"Architected" is actually the intended word (and is an actual word!).  I'll merge your PR and revert this.

![shot](https://user-images.githubusercontent.com/2362137/113636786-4ed25e80-96c7-11eb-8ec2-970563bc1473.png)

Reply:
Sorry, I looked it up and missed that one.